hash/nes.xml

```diff
@@ -1022,6 +1022,25 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="aburnerj1" cloneof="aburner">
+		<description>After Burner II (Jpn, Alt PCB)</description>
```

Comment:
What do you mean by “Alt PCB” – you list it as having the same `SUNSOFT-4` PCB as the other one.  Also, don’t capitalise things that aren’t proper nouns or initialisims in disambiguation text in descriptions.

Reply:
Unless I'm wrong "pcb" values in nes.xml seem to be tracking bootgod's "PCB Class" field. It's listed as SUNSOFT-4 there, but PCB is one of several variants in that class. nes.xml doesn't seem to track all of the variant PCBs. Actual label: SUNSOFT-6A. Should I change the description to just "alt"? (I figured Alt PCB also better informed the user that this is an identical binary to aburnerj).

Reply:
I'm trying to be stylistically consistent here. "Alt PCB" is in 144 nes.xml descriptions. If I do "alt BCP" there would be 1 like that. I'm willing to change them all if it helps.

Reply:
Can you just change the PCB to what it actually is?  There‘s no point aligning with bootgod if it’s not actually preserving detail.

It would be better if software lists followed the conventions we use in MAME itself more closely.  There’s no need to update the entire software list in this pull request, but please follow recommendations for additions.

Reply:
For what it's worth bootgod's DB has the info, but the SL copying of it is hit and miss.

I've completed all the requested changes below. Sorry if I created more work for you—was trying to avoid that very thing by being thorough. Thanks for the reviewing.

Reply:
Doing to pre-commit check now and there is what appears to be one copy/paste mistake in the hash fields (SHA1) of an entry.  I didn't notice any other formatting errors.


In XML:
software name="f1senstnj" cloneof="f1senstn"
rom name="f-1 sensation (japan).chr" size="131072" crc="67853884" sha1="94f27858c002745b9666d36485526a6404b9db84" offset="00000" status="baddump"

Actual Image:
Name:	f1senstnj\f-1 sensation (japan).chr
Size:	131072
CRC32:	67853884
SHA1:	4a170b3dcf365badd2bea6984d0be84651459348

Reply:
Apologies. Thought I caught all of the checksum mistakes when I tested. Fixed.